hoot-core/src/main/cpp/hoot/core/conflate/network/DebugNetworkMapCreator.cpp

```diff
@@ -60,7 +60,7 @@ void DebugNetworkMapCreator::_addEdgeLink(OsmMapPtr map, NetworkEdgeScorePtr edg
     ConstNodePtr n1 = _getMedianNode(map, edgeScore->getEdgeMatch()->getString1()->getMembers());
     ConstNodePtr n2 = _getMedianNode(map, edgeScore->getEdgeMatch()->getString2()->getMembers());
 
-    WayPtr w(new Way(Status::Invalid, map->createNextWayId(), -1));
+    WayPtr w(new Way(Status::Invalid, map->createNextWayId(), ElementData::CIRCULAR_ERROR_EMPTY));
```

Comment:
`CIRCULAR_ERROR_EMPTY` is dropped into the node/way/relation indicating that it wasn't specified in the input data.